Editor/UI/Wizard/WizardPage.cs

```diff
@@ -23,6 +23,9 @@ public abstract class WizardPage
         [SerializeField]
         public bool CanProceed = true;
 
+        [SerializeField]
```

Comment:
Is the attribute necessary? 🤔
Since the class is already serializable and the field is public I think it is redundant.